src/components/Sidebar/UserDetails.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'; // we need this to make JSX compile
+import {
+    globalCss
+} from '../../styles/stitches.config';
+import userIcon from './img/userIcon.png';
+
+type CardProps = {
+    name: string,
+}
+
+const Avatar = () => <aside>
+    <div style = {{
```

Comment:
Can you write these using stitches?
Docs: https://stitches.dev/
Example: https://github.com/jakezachariahnixon/todo-app/tree/main/src/components/Badge

Reply:
Please check it for me. Thank you ! @jakezachariahnixon